### PR TITLE
Fix packaging for dbt `py` modules

### PR DESCRIPTION
**Pull request by wslulciuc**
This PR fixes the way the `py` modules for DBT are versioned to align more with the way existing `py` modules are released / versioned.

Thanks @collado-mike for the suggestion on the fix :)

### CODE REVIEW

File: integrations/dbt/bigquery/setup.cfg [hunk 1]
```diff
@@ -11,14 +11,14 @@ serialize =
 search = version="{current_version}"
 replace = version="{new_version}"
 
-[bumpversion:file:./setup.py]
-search = marquez-integration-common=={current_version}
```
Comment (collado-mike) [inline integrations/dbt/bigquery/setup.cfg:15]: Does this module not require the `marquez-integration-common` package?

Comment (wslulciuc) [reply integrations/dbt/bigquery/setup.cfg:15]: No, it was copied over by mistake. Here are the dependencies in [`setup.py`](https://github.com/MarquezProject/marquez/blob/fix/versioning-for-dbt-integrations/integrations/dbt/bigquery/setup.py):

```
requirements = [
    "dbt-core>=0.20.0b1",
    "dbt-bigquery>=0.20.0b1",
    "sqlparse>=0.3.1",
    "openlineage-python==0.0.1rc6"
]
```

**Review by collado-mike — APPROVED** [review]

File: integrations/dbt/bigquery/setup.cfg (hunk 1 above)
Comment (collado-mike) [reply integrations/dbt/bigquery/setup.cfg:15]: I see. I guess the "reusable" bigquery utilities aren't that reusable, then?

Comment (wslulciuc) [reply integrations/dbt/bigquery/setup.cfg:15]: That was my initial thinking as well. But, with the DBT integration being experimental, and using _adopters_ it most likely differs greatly from our airflow integration. Perhaps @mobuchowski could provide more insight?

**Pull request merged**